dimod/sampleset.py

```diff
@@ -997,6 +1002,90 @@ def aggregate(self):
 
         return type(self)(record, self.variables, self.info, self.vartype)
 
+    def append_variables(self, samples_like, sort_labels=True):
+        """Create a new sampleset with the given variables with values added.
+
+        Empty sample sets cannot be appended to.
+
+        Args:
+            samples_like:
+                Samples to add to the sample set. Should either be a single
+                sample or should match the length of the sample set. See
+                :func:`.as_samples` for what is allowed to be `samples_like`.
+
+            sort_labels (bool, optional, default=True):
+                If true, returned :attr:`.SampleSet.variables` will be in
+                sorted-order. Note that mixed types are not sortable in which
+                case the given order will be maintained.
+
+        Returns:
+            :obj:`.SampleSet`: A new sample set with the samples appended. If
```

Comment:
Again, when I read "samples appended" - the only possible interpretation of that in my head is samples are appended vertically. While, here, samples are actually *combined* on a sample-by-sample basis.

Reply:
Will change the language

Reply:
agree with @randomir here